lib/src/speed_dial.dart

```diff
@@ -217,23 +221,44 @@ class _SpeedDialState extends State<SpeedDial> with SingleTickerProviderStateMix
       curve: widget.curve,
     );
 
-    return Positioned(
-      bottom: widget.marginBottom - 16,
-      right: widget.marginRight - 16,
-      child: Container(
-        child: Column(
-          mainAxisAlignment: MainAxisAlignment.end,
-          crossAxisAlignment: CrossAxisAlignment.end,
-          children: List.from(fabChildren)
-            ..add(
-              Container(
-                margin: EdgeInsets.only(top: 8.0, right: 2.0),
-                child: animatedFloatingButton,
-              ),
+    switch (widget.orientation) {
+      case SpeedDialOrientation.Up:
```

Comment:
This should be also the default case for the switch, otherwise setting the orientation as null breaks everything.

Reply:
Done.